libs/expo/betterangels/src/lib/apollo/graphql/__generated__/types.ts

```diff
@@ -289,7 +289,7 @@ export type CreateNoteInput = {
   client?: InputMaybe<Scalars['ID']['input']>;
   privateDetails?: InputMaybe<Scalars['String']['input']>;
   publicDetails?: InputMaybe<Scalars['String']['input']>;
-  purpose: Scalars['String']['input'];
+  purpose?: InputMaybe<Scalars['String']['input']>;
```

Comment:
@mikefeldberg  - would you know if this is related to your comments in PR #607 about purpose becoming optional?
This ended up in my PR although it probably shouldn't have and is failing linting jobs. Or did I do something weird with my gql type generation?